packages/dropdowns/src/elements/Multiselect/Multiselect.tsx

```diff
@@ -32,34 +32,43 @@ import { REMOVE_ITEM_STATE_TYPE } from '../Dropdown/Dropdown';
 interface IMultiselectProps extends HTMLAttributes<HTMLDivElement> {
   /** Applies compact styling */
   isCompact?: boolean;
-  /** Removes borders and padding */
+  /** Removes the borders and padding */
   isBare?: boolean;
-  /** Indicates that the element is not interactive */
+  /** Disables the element's interactivity */
   disabled?: boolean;
-  /** Applies inset `box-shadow` styling on focus */
+  /** Applies `box-shadow` styling on focus */
   focusInset?: boolean;
-  /** Indicates that the element's menu is open */
+  /** Sets the element's menu to open */
   isOpen?: boolean;
-  /** Defines text that appears in the element when no items are selected */
+  /** Sets the placeholder text which appears when no items are selected */
   placeholder?: string;
-  /** Defines the element's validation state */
+  /** Sets the the element's validation state */
   validation?: VALIDATION;
-  /** Determines the maximum number of items displayed while collapsed */
+  /** Sets the maximum number of items displayed when collapsed */
   maxItems?: number;
-  /** Callback that overrides the "+ N more" text displayed when the total number of items exceeds `maxItems` */
+  /**
+   * Handles when more items are shown during expansion, and updates the number value in the "+ NUMBER more" text.
+   * This callback only applies when the total number of items exceeds `maxItems`.
+   *
+   * @param {number} index The index being expanded
+   * @return {string}
+   */
   renderShowMore?: (index: number) => string;
-  /** Callback that renders each item element */
+  /**
+   * Handles rendering for each item
+   *
+   * @param {value: any; removeValue: () => void} options The index being expanded
+   * @return {React.ReactElement}
+   */
   renderItem: (options: { value: any; removeValue: () => void }) => React.ReactElement;
```

Comment:
Please note the following format for documenting objects in JSDoc:

```
/**
 * Overall description ...
 *
 * @param {[options.value]: any} description...
 * @param {[options.removeValue]: () => void} description...
 * @return {React.ReactElement} description...
 */
```

@austingreendev please add context here by providing suggested text for the missing descriptions seen in my formatted comment block.